src/actions/RecordsActions.js

```diff
@@ -61,9 +61,6 @@ export function exportRecords(exportType, params = {}) {
       .get(`${API_URL}/rest/records/export`, {
         params,
         paramsSerializer,
-        headers: {
```

Comment:
I removed this as described here: https://github.com/kbss-cvut/record-manager/pull/52#discussion_r1672808403

Reply:
I suggest to revert this change and keep the accept header.
Without it the backend will not receive the export type and it will always return JSON.

Reply:
No, it returns the correct file type even without it (not sure how tbh, I think it it because of the file extension), but if I add the headers, the backend returns 406 (only for excel file types)

Reply:
Oohh ok I see, then I can put back the headers but I will get 406 error.

Reply:
@kostobog I reverted

Reply:
@LaChope @blcham 
Export of both json and excel in distribution works for small number of records. I tested it with 71 records.